tap_bing_ads/__init__.py

```diff
@@ -61,9 +61,11 @@ class InvalidDateRangeEnd(Exception):
     pass
```

Comment:
@dbshah1212 We also have `exclusions.py` and `reports.py` in this tap. Can you add comments in those files as well?

Reply:
@karanpanchal-crest `exclusions.py` and `reports.py` contain only constant with comments and there is no logical code defined in this.